package.json

```diff
@@ -80,17 +79,20 @@
     "lru-cache": "^6.0.0",
     "stanza": "^12.13.1",
     "superagent": "^5.1.0",
+    "unorm": "^1.6.0",
```

Comment:
[unorm](https://github.com/walling/unorm/) is a little outdated, but we only need it to make normalization work in IE which is also outdated.